src/bluefinClient.ts

```diff
@@ -277,11 +280,12 @@ export class BluefinClient {
     const keyPair = getKeyPairFromPvtKey(_account, "ZkLogin");
     this.signer = keyPair;
     this.walletAddress = walletAddress;
-    this.isZkLogin = true;
     this.maxEpoch = maxEpoch;
     this.decodedJWT = decodedJWT;
     this.proof = proof;
     this.salt = salt;
+    this.isZkLogin = true;
+    this.is_wallet_extension = false;
```

Comment:
Is it required given that by default its already false ?

Reply:
That's an intended change. Right now we've three initializers

1. initalizeWithKeyPair (not for UI)
2. initializeWithHook (when using wallet extension on UI)
3. initializeForZkLogin (when signing with google on UI)

so, 

for initializeWithHook -> value for zk_login should be set to false and the value for is_wallet_extension should be set to true as these values are being used to determine logic

for initializeForZkLogin -> value for zk_login should be set to true and the value for is_wallet_extension should be set to false as these values are being used to determine logic